dts/bindings/misc/skyworks,sky13317.yaml

```diff
@@ -11,4 +12,6 @@ properties:
   gpios:
     type: phandle-array
     required: true
-    description: Antenna mux control pins
+    description: Antenna mux control pins. Length 1-3
```

Comment:
Is there a straight-forward way to include both the muxes and the gpios themselves?

Reply:
I am not sure why there would be. They are completely different things. Pinctrl is for configuring the pins (input output, I2C, etc).
Gpios is to define GPIO pins specifically to make available for use from driver.

Reply:
For reference: https://docs.zephyrproject.org/latest/hardware/pinctrl/index.html#pin-control-vs-gpio